posthog/hogql/database/test/__snapshots__/test_database.ambr

```diff
@@ -1,6 +1,6 @@
 # serializer version: 1
 # name: TestDatabase.test_database_warehouse_joins_persons_poe_v2
-  'SELECT events__some_field.key AS key FROM events LEFT JOIN (SELECT groups.key AS key, key AS events__some_field___key FROM (SELECT groups.group_type_index AS index, groups.group_key AS key FROM groups WHERE equals(groups.team_id, 226) GROUP BY groups.group_type_index, groups.group_key) AS groups) AS events__some_field ON equals(replaceRegexpAll(nullIf(nullIf(JSONExtractRaw(events.person_properties, %(hogql_val_0)s), \'\'), \'null\'), \'^"|"$\', \'\'), events__some_field.events__some_field___key) WHERE equals(events.team_id, 226) LIMIT 10000'
+  'SELECT events__some_field.key AS key FROM events LEFT JOIN (SELECT groups.key AS key, key AS events__some_field___key FROM (SELECT groups.group_type_index AS index, groups.group_key AS key FROM groups WHERE equals(groups.team_id, 225) GROUP BY groups.group_type_index, groups.group_key) AS groups) AS events__some_field ON equals(replaceRegexpAll(nullIf(nullIf(JSONExtractRaw(events.person_properties, %(hogql_val_0)s), \'\'), \'null\'), \'^"|"$\', \'\'), events__some_field.events__some_field___key) WHERE equals(events.team_id, 225) LIMIT 10000'
```

Comment:
this team id should be being replaced or this test will flap 🙈 

(looks totally separate to this though)

Reply:
getting fixed.. soon